pgmpy/estimators/MirrorDescentEstimator.py

```diff
@@ -45,7 +46,7 @@ def _calibrate(self, theta, n):
         log_z = logsumexp(mu[clique].values)
         for clique in cliques:
             mu[clique] += np.log(n) - log_z
```

Comment:
@jakee417 This should be `mu[clique].values` ?

Reply:
`mu[clique]` is a `DiscreteFactor` which I believe implements addition by a `float`.